.yarnrc.yml

```diff
@@ -1,10 +1,11 @@
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
-    spec: '@yarnpkg/plugin-interactive-tools'
+compressionLevel: mixed
 
-yarnPath: .yarn/releases/yarn-3.4.1.cjs
+enableGlobalCache: false
 
-npmAuthToken: ${NODE_AUTH_TOKEN:-}
 enableTelemetry: false
 
-nodeLinker: 'node-modules'
+nodeLinker: node-modules
+
+npmAuthToken: "${NODE_AUTH_TOKEN:-}"
+
+yarnPath: .yarn/releases/yarn-4.0.2.cjs
```

Comment:
I'll test as well. What node version are you on?

Reply:
I'm on node 21.4

Reply:
I'm on the LTS version, and it did install yarn 1 for me as well. 
I'll look into it

Reply:
I found the issue, we need to run `corepack enable` for Corepack to actually be on 🤦🏻 
Might as well use the file then, since it is for some reason turned off by default 🫠